### PR TITLE
CI: Add release workflow

### PR DESCRIPTION
**Pull request by SchoolGuy**
To spare me and @brejoc the work of doing the release by hand, I would love to use a GH Action to do that release. I will use an API token generated by myself to authenticate against PyPI.

### CODE REVIEW

**Comment by SchoolGuy** [comment]
Note: This is for now just the default file copied from the guide - https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

I will adjust it to our needs tomorrow.

**Comment by SchoolGuy** [comment]
I think this is ready to be reviewed.

File: .github/workflows/release_master.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  DEFAULTPATH: "etc/sysconfig"
+
+  APACHE_USER: "www-data"
+  HTTP_USER: "www-data"
+  APACHE_GROUP: "www-data"
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
```
Comment (nodeg) [inline .github/workflows/release_master.yml:31]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  DEFAULTPATH: "etc/sysconfig"
+
+  APACHE_USER: "www-data"
+  HTTP_USER: "www-data"
+  APACHE_GROUP: "www-data"
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
```
Comment (nodeg) [inline .github/workflows/release.yml:31]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```
Comment (nodeg) [inline .github/workflows/release.yml:37]:
```suggestion
      - uses: actions/checkout@v2
      - name: Set up Python 3.9
        uses: actions/setup-python@v2
        with:
          python-version: 3.9
```
Use a more recent version of Python, [setup-python V2](https://github.com/actions/setup-python) and [checkout V2](https://github.com/actions/checkout).

File: .github/workflows/release_master.yml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```
Comment (nodeg) [inline .github/workflows/release_master.yml:37]:
```suggestion
      - uses: actions/checkout@v2
      - name: Set up Python 3.9
        uses: actions/setup-python@v2
        with:
          python-version: 3.9
```
Use a more recent version of Python, [setup-python V2](https://github.com/actions/setup-python) and [checkout V2](https://github.com/actions/checkout).

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+          fakeroot \
+          patch \
+          pax \
+          git \
+          hardlink \
+          apache2 \
+          libapache2-mod-wsgi-py3 \
+          systemd
+      - name: Install pypa/build
+        run: python -m pip install build --user
```
Comment (nodeg) [inline .github/workflows/release.yml:85]:
```suggestion
        run: python -m pip install --upgrade pip
        run: python -m pip install build --user
```

File: .github/workflows/release_master.yml [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+          fakeroot \
+          patch \
+          pax \
+          git \
+          hardlink \
+          apache2 \
+          libapache2-mod-wsgi-py3 \
+          systemd
+      - name: Install pypa/build
+        run: python -m pip install build --user
```
Comment (nodeg) [inline .github/workflows/release_master.yml:85]:
```suggestion
        run: python -m pip install --upgrade pip
        run: python -m pip install build --user
```

File: .github/workflows/release.yml (hunk 5 above)
Comment (nodeg) [reply .github/workflows/release.yml:85]: Could be useful to upgrade pip here.

**Review by nodeg — APPROVED** [review]
Some suggestions, but not necessary to adjust. Otherwise LGTM.

**Comment by SchoolGuy** [comment]
@nodeg Comments greatly appreciated!

**Comment by SchoolGuy** [comment]
Okay apparently the empty lines between the environment variables were an error. Thus now the workflow is valid. When the CI passes, I will merge this as we can't try this on a PR. Fixup PRs are bad but necessary here if my workflow fails.

**Pull request merged**